### PR TITLE
PRM-2783 added new hintext to pages

### PR DESCRIPTION
**Pull request by RoheelArshad**

### CODE REVIEW

File: app/views/reasonableExcuseJourneys/other/WhyReturnSubmittedLatePage.scala.html [hunk 1]
```diff
@@ -45,11 +46,14 @@
...
   @pageHeader()
 
-  @formHelper(action = postAction) {
+@formHelper(action = postAction) {
     @characterCount(
       form("why-return-submitted-late-text"),
       label = messages(pageTitleAndHeading),
-      hint = None,
+      hint = if(appealType.contains(PenaltyTypeEnum.Late_Payment) || appealType.contains(PenaltyTypeEnum.Additional))
+Some(HtmlContent(messages("otherReason.whyReturnSubmittedLate.lpp.p1")))
```
Comment (LewisMitchell) [inline app/views/reasonableExcuseJourneys/other/WhyReturnSubmittedLatePage.scala.html:53]: `otherReason.whyReturnSubmittedLate.p1` should be shown when the user is appealing multiple penalties. Not just an LPP. I believe there is a session key to figure this out: https://github.com/hmrc/penalties-appeals-frontend/blob/563f6bbbcd2c6661ff500e3e261d166ae2010c0c/app/utils/SessionKeys.scala#L57

**Review by LewisMitchell — CHANGES_REQUESTED** [review]

File: conf/messages [hunk 2]
```diff
@@ -295,6 +295,8 @@ otherReason.whyReturnSubmittedLate.lpp.headingAndTitle = Why was the VAT bill pa
 otherReason.whyReturnSubmittedLate.lpp.error.required = Explain why the VAT bill was paid late
 otherReason.whyReturnSubmittedLate.error.required = Explain why the return was submitted late
 otherReason.whyReturnSubmittedLate.error.regex = The text must contain only letters, numbers and standard special characters
+otherReason.whyReturnSubmittedLate.p1 = We only need to know about these penalties. Appeal other penalties separately.
+otherReason.whyReturnSubmittedLate.lpp.p1 = We only need to know about this penalty. Appeal other penalties separately.
```
Comment (LewisMitchell) [inline conf/messages:299]: The naming of the messages should change slightly:

- 'We only need to know about these penalties' refers to multiple penalties
- 'We only need to know about this penalty' refers to a single penalty

**Review by LewisMitchell — CHANGES_REQUESTED** [review]

File: app/views/reasonableExcuseJourneys/other/WhyReturnSubmittedLatePage.scala.html (hunk 1 above)
Comment (RoheelArshad) [reply app/views/reasonableExcuseJourneys/other/WhyReturnSubmittedLatePage.scala.html]: updated

File: conf/messages (hunk 2 above)
Comment (RoheelArshad) [reply conf/messages]: updated

**Review by LewisMitchell — APPROVED** [review]

**Pull request merged**